src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -154,46 +157,46 @@ def _create_blocks(
         Returns:
             List of slack blocks.
         """
-        if (
-            isinstance(params, SlackAlerterParameters)
-            and hasattr(params, "payload")
-            and params.payload is not None
-        ):
-            payload = params.payload
-            return [
-                {
-                    "type": "section",
-                    "fields": [
-                        {
-                            "type": "mrkdwn",
-                            "text": f":star: *Pipeline:*\n{payload.pipeline_name}",
+        if isinstance(params, SlackAlerterParameters):
+            if hasattr(params, "blocks") and params.blocks is not None:
```

Comment:
I would consider adding `logger.info()` blocks to all conditions to tell the user what is happening

Reply:
Logs added for each step - let me know if you want me to be more verbose in the logging though 👍